backends/concrete-cuda/implementation/test_and_benchmark/test/test_bit_extraction.cpp

```diff
@@ -161,10 +195,30 @@ ::testing::internal::ParamGenerator<BitExtractionTestParams>
         // n, k, N, lwe_variance, glwe_variance, pbs_base_log, pbs_level,
         // ks_base_log, ks_level, number_of_message_bits,
         // number_of_bits_to_extract, number_of_inputs
-        (BitExtractionTestParams){585, 1, 1024, 7.52316384526264e-37,
-                                  7.52316384526264e-37, 10, 2, 4, 7, 5, 5, 1},
-        (BitExtractionTestParams){481, 1, 1024, 7.52316384526264e-37,
-                                  7.52316384526264e-37, 4, 7, 1, 9, 5, 5, 1});
+        (BitExtractionTestParams){585,
+                                  1,
+                                  1024,
+                                  7.52316384526264e-37,
+                                  7.52316384526264e-37,
+                                  10,
+                                  2,
+                                  4,
+                                  7,
+                                  {5, 4, 4, 3},
+                                  {5, 4, 4, 3},
+                                  4},
+        (BitExtractionTestParams){481,
```

Comment:
I think we could remove this second parameter set, it's very similar to the first one.